src/Gui/NavigationStyle.cpp

```diff
@@ -1479,8 +1479,10 @@ SbBool NavigationStyle::processEvent(const SoEvent * const ev)
     // check for left click without selecting something
     if ((curmode == NavigationStyle::SELECTION || curmode == NavigationStyle::IDLE)
             && !processed) {
-        if (SoMouseButtonEvent::isButtonReleaseEvent(ev,SoMouseButtonEvent::BUTTON1))
-            Gui::Selection().clearSelection();
+        if (SoMouseButtonEvent::isButtonReleaseEvent(ev, SoMouseButtonEvent::BUTTON1)) {
+            if (!(QApplication::keyboardModifiers() == Qt::ControlModifier))
```

Comment:
@PaddleStroke I think this is the wrong check. If you additionally press another key like Shift or Alt then it still clears the selection. Btw you can also use Inventor API: `if (!ev->wasCtrlDown())`

Reply:
Seconds @wwmayer here using Inventor API for code consistency.
Also the 2 'if' can be joigned together with `&&` for clarity.